src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala

```diff
@@ -12,7 +12,7 @@ object AnnualUpdateMinimalDF {
     "is__birdlife_alliance_for_zero_extinction_sites",
     "gfw_planted_forests__type",
     "is__gmw_global_mangrove_extent_1996",
-    "is__gmw_global_mangrove_extent_2016",
```

Comment:
I think you should check with the flagship frontend that this name change won't break them, since they might use this name in the widget. I know they use this field for GLAD and VIIRS, but I'd double check if they even use it for TCL (since TCL goes back to 2000). If not, we could just get rid of it.

I'm also not sure if we want to change it to latest, because then it becomes unclear what year the extent is from when you read the table, and I know we've had confusion about that in the past. It might just be better to explicitly include 2020 and update it whenever it changes every 3-4 years. But less sure about that.

Reply:
Changed `latest` to `2020` in latest commit. I'll let Flagship now before handoff